js/hoot/control/utilities/Folder.js

```diff
@@ -139,6 +139,32 @@ Hoot.control.utilities.folder = function(context) {
             });
         }
 
+        function updateLastAccessed(node) {
+            var row = d3.select(node.parentNode);
+            row.classed('expiring', true)
+                .append('g').append('svg:foreignObject')
+                .attr('class', 'expiring')
+                .attr('transform', function(d) {
+                    var dd = d.depth-1;
+                    var dy=772+(11*dd);
+                    return 'translate('+ dy +',-9)'; 
+                });
+            var tooltip = d3.select('#datasettable')
+                .append('div')
```

Comment:
Seems like the tooltip div should only get appended once.  This will append a new div every time the method is called.  You can fix that by adding a data binding of `.data([0]).enter()` so that there will only be one tooltip.

Reply:
I couldn't get this method to work.